CMakeLists.txt

```diff
@@ -41,11 +41,20 @@ project(tritonpytorchbackend LANGUAGES C CXX)
 #   - Set TRITON_PYTORCH_DOCKER_IMAGE to use the docker image of
 #     PyTorch to base the build off.
 #
+# TRITON_PYTORCH_ENABLE_TORCHTRT: enables Torch-TRT runtime support.
+#   With TRITON_PYTORCH_DOCKER_IMAGE: set if to OFF for images before 21.11
+#   Without TRITON_PYTORCH_DOCKER_IMAGE:
+#     If you leave it ON: set TRITON_PYTORCH_TORCHTRT_LIB_PATH to Torch-TRT runtime lib. 
```

Comment:
@CoderHam seems like the jetson build will need some updates for this.

Reply:
The general recommendation for people building a specific version of Triton is to use the corresponding release branch. I would suggest not adding sections like 
```
#   With TRITON_PYTORCH_DOCKER_IMAGE: set if to OFF for images before 21.11
```
Similarly for TRITON_PYTORCH_TORCHTRT_LIB_PATH, we want to follow a part like we do for TRITON_PYTORCH_LIB_PATHS. In the case where the TRITON_PYTORCH_LIB_PATHS is specified, we set TRITON_PYTORCH_DOCKER_BUILD to ON.

When TRITON_PYTORCH_TORCHTRT_LIB_PATH is specified we should mandate the specification of TRITON_PYTORCH_LIB_PATHS and turn Docker build to be ON.

Reply:
Coming back to Jetson build, yes we will need to pass TRITON_PYTORCH_TORCHTRT_LIB_PATH to build.py in our jetson_build.sh script.

Currently we use a single arg called for library paths to pass these paths to build.py 

My suggestion is to keep a single argument for TRITON_PYTORCH_LIB_PATHS where the paths are specified in a `;` separated fashion as is the norm.

Reply:
Yes I agree single TRITON_PYTORCH_LIB_PATHS is the best way to go here.

Reply:
It also seems, for integration purposes, I should set default for TRITON_PYTORCH_ENABLE_TORCHTRT to OFF. Maybe later we can flip it.

Reply:
I don't see why Jetson would need to handle that differently - PyTorch docker image for Jetson will have TorchTRT in.

Reply:
@deadeyegoodwin do we plan to enable TRITON_PYTORCH_ENABLE_TORCHTRT by default in 20.11?

Reply:
> I don't see why Jetson would need to handle that differently - PyTorch docker image for Jetson will have TorchTRT in.

For jetson, we do not use docker for our build. Currently we do not officially support Pytorch on Jetson but when we do it will be require us to copy the respective libraries / source code to a local dir and link against them instead of using docker like we do on x86.

In the long term future we might want to dockerize our Jetson build but till then we are limited as above.

Reply:
> 
> 
> @deadeyegoodwin do we plan to enable TRITON_PYTORCH_ENABLE_TORCHTRT by default in 20.11?

21.12? 21.11 is done. We can make it default in 21.12 if we have sufficient testing. Is there a reason not to? We can turn it off for jetson if that is necessary.

Reply:
Yes 21.12. Understood, we can set the default to ON once we have sufficient testing. I will make a note of the same.

Reply:
I have made it automatic (Cmake configured based on presence of Torch-TRT in PyTorch image).

Reply:
Right now OFF is a default setting. How do I tell default OFF from user-specified ?

Reply:
Reworked the default: now it's ON, and user can set it to OFF and it will stay off.
If set/left to ON, it would be disabled if the image does not have Torch-TRT, or if the local lib path does not contain installed Tocrh-TRT package location.

Reply:
Yes that should be good for now